pdbtools/pdb_uniqname.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 João Pedro Rodrigues
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Renames atoms sequentially (C1, C2, O1, ...) for each HETATM residue.
+
+Relies on an element column being present (see pdb_element).
+
+Usage:
+    python pdb_uniqname.py -<option> <pdb file>
```

Comment:
In  #71, it was decided the name of the new tool would be `pdb_rename`. Here you propose `pdb_uniqname`.

I like `pdb_uniqname`, it shows the purpose of a protocol rather than an user-driven intention as with `pdb_rename` (*rename name*) , IMHO,

Reply:
I know :) I decided for `uniqname` because it's clearer. `rename` could mean so many things...